app/assets/javascripts/initializers/initializeHeroBannerClose.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+function initializeHeroBannerClose() {
+  let banner = document.getElementById("js-she-coded-hero-banner");
+  let closeIcon = document.getElementById("js-she-coded-hero-banner__x");
+
+  if (banner && closeIcon) {
+    closeIcon.onclick = (e) => {
+      document.cookie = "heroBanner=false";
```

Comment:
We wouldn't want the cookie to live on forever, hence we would want to do one of the following: 
1.  On logout we can remove the cookie, this will reset the banner and show it when a user signs in again
2. Toggle to delete the cookies after a couple of days (through campaign config panel)
3. Have a setting on the config panel that allows the user to specify after how many days it will expire 

I need to research the options and I will tackle this in a subsequent PR.

Reply:
> Have a setting on the config panel that allows the user to specify after how many days it will expire

I love this as the strategy for expiring campaigns in the long run 👍 But, setting a specific expiration date post-campaign seems good for now!